Templates/QueueTrigger-Powershell/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "defaultFunctionName": "QueueTriggerPowershell",
+    "defaultFunctionName": "QueueTriggerPowerShell",
     "description": "A PowerShell function that will be run whenever a message is added to a specified Azure Storage queue",
     "name": "QueueTrigger - PowerShell",
-    "language": "Powershell",
+    "language": "PowerShell",
```

Comment:
This casing change should be done in all powershell templates - metadata.json should be the same. Otherwise two "powershell" options would show up in the portal.